src/main/java/com/github/manolo8/darkbot/config/ColorScheme.java

```diff
@@ -65,7 +65,7 @@ public static class SafetyEditor {
     public static class Fonts {
         public @Option Font BIG = new Font("Consolas", Font.PLAIN, 32);
         public @Option Font MID = new Font(Font.SANS_SERIF, Font.PLAIN, 18);
-        public @Option Font SMALL = new Font("Consolas", Font.PLAIN, 12);
+        public @Option Font SMALL = new Font("Consolas", Font.PLAIN, 13);
```

Comment:
Sadly this change is not one that users will get if you don't make specific logic for that. Luckily you have already needed to make changes to the format, and FontAdapter could make it so if you deserialize, with 2 fields, 13:consolas, turns it into 13:consolas:plain